src/main/java/ee/tuleva/onboarding/mandate/payment/rate/PaymentRateValidator.java

```diff
@@ -0,0 +1,21 @@
+package ee.tuleva.onboarding.mandate.payment.rate;
```

Comment:
I'd call this package paymentrate instead of payment.rate (which would imply that rate is a sub-package of payment)